process/charter.html

```diff
@@ -81,189 +77,188 @@ <h4>Tools and templates</h4>
-The Strategy or Project Lead (or delegate) informs the W3C Communications Team of TiLT approval
-by sending email to <a
-href="mailto:w3t-comm@w3.org">w3t-comm@w3.org</a>.</p>
+<h3 id="recharter-request">Request for rechartering</h3>
 
-<p class="timeline"><em>Timeline:</em> this takes 2 weeks.</p>
+<p>A group recharters when it wishes to change its charter or extend it longer than six months.</p>
 
-  <h3 id="extension">Charter Extension and Update</h3>
+<p>In these processes, the roles of the <a href="#charter-shepherd">charter shepherd</a> include:</p>
 
-<p><strong>Note:</strong> Is the Group aware that its charter is being extended? Please inform the group before discussion within TiLT.</p>
+<dl>
+  <dt>Record group decision</dt>
+  <dd>The group should discuss the situation and record a formal decision to request extension or rechartering.</dd>
+  <dt>Follow process for creating a charter</dt>
+  <dd>Rechartering is handled almost the same as <a href="#new-charters">new charters</a>. Not that, in addition to any content changes, the charter may need to be updated if the <a href="https://w3c.github.io/charter-drafts/charter-template.html">charter template</a> has changed. Furthermore, the template tool will prompt for the inclusion of Patent Policy language and otherwise help you meet the
+	<a href="https://www.w3.org/Consortium/Process/#WGCharter">list of charter requirements in the Process</a>. For existing groups, the <a href="https://www.w3.org/groups/wg/css/deliverables">charter assistant</a> helps in producing the list of exclusion drafts.
+</dd>
+</dl>
 
-<p>W3C resolved to adopt the <a href="https://www.w3.org/2015/04/charter-extensions.html">Policy on W3C Group Charter End Dates</a>, effective 18 June 2015.</p>
+<h2 id="implementation-details">Implementation Details</h2>
 
-<p>W3M amended the policy on 17 November 2021: charter extension can only be up to 6 months beyond the end date of the current charter. Any request for more than 6 months
-  MUST treated as <a href='#creation'>rechartering</a>.</p>
+<p>The following sections are mostly intended as instructions to the Team and are included here for transparency.</p>
 
-<p>TiLT keeps following those resolutions.</p>
+<h3 id="baseline-support">Sufficient Member support for a charter</h3>
 
-<p>If TiLT resolves to extend a charter, the Strategy or Project Lead (or delegate) sends e-mail to the
-W3C Communications Team in the form of an extension announcement; see the
-<a href="https://www.w3.org/new-doc-from-template?location=%2FTeam%2Ftemplates&amp;template=%2Fafs%2Fw3.org%2Fpub%2FWWW%2FTeam%2FTemplates%2Fcharter-extension.html&amp;submit=Continue...">charter extension template</a>.</p>
+<p><a href="https://lists.w3.org/Archives/Member/w3c-ac-members/2014JulSep/0018.html">Since July 2014</a>, the Team seeks a baseline level of support from Members before approving a charter.</p>
```

Comment:
At this point, I think this notice can be removed. No need to link to the reason why a paragraph is there.

Reply:
Done.